src/states/elder/mod.rs

```diff
@@ -1349,6 +1355,15 @@ impl Approved for Elder {
 
         self.send_routing_message(RoutingMessage { src, dst, content })
     }
+
+    fn handle_user_event(
+        &mut self,
+        payload: Vec<u8>,
+        outbox: &mut dyn EventBox,
+    ) -> Result<(), RoutingError> {
+        self.send_event(Event::Consensus(payload), outbox);
+        Ok(())
+    }
```

Comment:
The implementations are the same in both `Adult` and `Elder` - wouldn't it be better to make this the default implementation in the `Approved` trait?

Reply:
Agreed, good suggestion :+1: